gtk4/src/subclass/root.rs

```diff
@@ -1,6 +1,6 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use crate::subclass::prelude::NativeImpl;
+use crate::subclass::prelude::*;
 use crate::Root;
 use glib::subclass::prelude::*;
```

Comment:
Should already be available from `use crate::subclass::prelude::*;`. Do check the other files, too :)